examples/correlation_alignment_example.py

```diff
@@ -105,7 +105,7 @@
         data.ift(list(signal_pathway.keys()))
         opt_shift, sigma, mask_func = psdpr.correl_align(
             data * mysgn,
-            indirect_dim=indirect,
+            repeat_dims=[indirect],
```

Comment:
this doesn't make sense.  This should default to an empty list.

Reply:
are you saying don't feed a repeat_dims argument? - mind, this is the example